.env.test

```
@@ -1 +1,37 @@
+REDIS_URL=redis://localhost:6379
+
+# 그래프 DB
+NEO4J_URI=bolt://localhost:7687
+NEO4J_USER=neo4j
+NEO4J_PASSWORD=test_password
+GRAPH_SCORE_WEIGHT=0.3
+
+# AI & Search
+OPENAI_API_KEY=sk-test-key-placeholder
 TAVILY_API_KEY=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
+EMBEDDINGS_PROVIDER=openai
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EMBEDDINGS_PROVIDER key should go before the OPENAI_API_KEY key

(UnorderedKey)